runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/Go.test.stg

```diff
@@ -18,7 +18,16 @@ AppendStr(a,b) ::= "<a> + <b>"
 
 Concat(a,b) ::= "<a><b>"
 
-AssertIsList(v) ::= "TODO!!"
+AssertIsList(v) ::= <<
+// A noddy range over the list will not compile if it is not getting a slice
+// however, Go will not compile the generated code if the slice vs single value is wrong.
+// Makes the Java based tests suite work though.
```

Comment:
Not sure comments should be presented in generated code thus they should not be presented in template.

Reply:
Not sure why that matters? It is only generated for the test. I have found the generated test code wanting - a few comments would have helped understand what was going on much more quickly.

Reply:
@parrt @KvanTTT You guys want me to take out the generated comments before you merge it? No worries either way for me.

Reply:
We can just leave it since that's easier.

Reply:
Ok, nevermind.